src/app/clusters/level-control/level-control.cpp

```diff
@@ -595,7 +597,7 @@ static void moveHandler(uint8_t commandId, uint8_t moveMode, uint8_t rate, uint8
         goto send_default_response;
     }
 
-    if (!shouldExecuteIfOff(endpoint, commandId, optionMask, optionOverride))
+    if (rate == 0 || !shouldExecuteIfOff(endpoint, commandId, optionMask, optionOverride))
```

Comment:
So this just treats Move commands with a rate of 0 as no-ops?

This is not defined in the ZCL spec.  Could you get an issue filed to make sure this is defined in the CHIP data model?